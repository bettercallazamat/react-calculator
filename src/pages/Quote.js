const Quote = () => (
  <div>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt
      ut labore et dolore magna aliqua.
    </p>
    <p>
      â€” Lorem ipsum dolor sit amet
    </p>
  </div>
);

export default Quote;
