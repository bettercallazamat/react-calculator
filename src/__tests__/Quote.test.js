import React from 'react';
import { render } from '@testing-library/react';
import Quote from '../pages/Quote';

describe('Snapshot Home page', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<Quote />);
    expect(asFragment(<Quote />)).toMatchSnapshot();
  });
});
