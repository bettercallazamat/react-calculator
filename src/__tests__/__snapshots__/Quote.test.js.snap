// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Home page should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="quoteDiv"
  >
    <img
      alt="author"
      src="walle-john.jpeg"
    />
    <p>
      Mathematics is much more than computation with pencil and a paper and getting answers to routine exercises. In fact, it can easily be argued that computation, such as doing long division, is not mathematics at all. Calculators can do the same thing and calculators can only calculate â€“ they cannot do mathematics.
    </p>
    <p>
      - John A. Van de Walle
    </p>
  </div>
</DocumentFragment>
`;
